src/nav2_msgs/msg/Costmap.msg

```diff
@@ -0,0 +1,9 @@
+# This represents a 2-D grid map, in which each cell has an associated cost
+
+std_msgs/Header header
+
+# MetaData for the map
+CostmapMetaData info
+
+# The cost data, in row-major order, starting with (0,0).
+uint8[] data
```

Comment:
Seems like there needs to be newline here.

Reply:
Added